app.js

```diff
@@ -0,0 +1,34 @@
+// 1. IMPORTACIONES
+const express 		= require("express")
+const app			= express()
+
+require("dotenv").config()
+
+// 2. MIDDLEWARES
+app.use(express.static('public'))
+
+app.set("views", __dirname + "/views")
+app.set("view engine", "hbs")
+
+
+// 3. RUTAS
+app.get("/", (req, res) => {
+
+res.sendFile(__dirname + '/views/index.html')
+
+})
+app.get("/about", (req,res) => {
+    res.sendFile(__dirname + '/views/about.html')
+})
+
+app.get("/works", (req,res) => {
+    res.sendFile(__dirname + '/views/works.html')
+})
+
+app.get("/photo-gallery", (req,res) => {
+    res.sendFile(__dirname + '/views/photo-gallery.html')
+})
+// 4. SERVIDOR
+app.listen(process.env.PORT, () => {
+	console.log(`Servidor activo en puerto ${process.env.PORT}`)
```

Comment:
Con ayuda de la siguiente linea y ctr o cmd + click puedes abrir la ruta desde tu terminal, esto va a ahorrar tiempo en el desarrollo de tus proyectos 😉 
```suggestion
    console.log(`Servidor activado en el puerto http://localhost:${process.env.PORT}`)
```